priv/templates/phoenix.gen.presence/presence.ex

```diff
@@ -0,0 +1,66 @@
+defmodule <%= module %> do
+  @moduledoc """
+  Provides presence tracking to channels and processes.
+
+  See the [`Phoenix.Presence`](http://hexdocs.pm/phoenix/Phoenix.Presence.html)
+  docs for more details.
+
+  ## Usage
+
+  Presences can be tracked in your channel after joining:
+
+      defmodule <%= base %>.MyChannel do
+        alias <%= base %>.Presence
```

Comment:
Let's include `use <%= base %>.Web, :channel`, the alias and get rid of the `...` below.